continuous_integration/environment-mindeps-array-dataframe.yaml

```diff
@@ -6,8 +6,8 @@ dependencies:
   - python=3.7
   - pyyaml
   # optional dependencies pulled in by pip install dask[array, dataframe]
-  - numpy=1.15
-  - pandas=0.23
```

Comment:
It wasn't clear to me whether these were intentionally pinned down so low. If unpinning is controversial, I can move this change out of this branch and try something different for the array-dataframe case

Reply:
I'm going to remove this change since it still failed anyways. I'll tackle in a separate PR.